examples/extract_mnist_cpu.py

```diff
@@ -12,6 +12,16 @@
 from privacyraven.models.victim import train_four_layer_mnist_victim
 from privacyraven.models.four_layer import FourLayerClassifier
 
+
+# PrivacyRaven's builtin victim functions allow the user to specify the number of GPUs
+# to use during computation via the `gpus` keyword argument.  By default, this is 1, but 
+# setting gpus=torch.cuda.device_count() will let PrivacyRaven / Pytorch use all available GPUs.
```

Comment:
I think putting this comment here is too much information for one of the starter examples.